src/app/index.test.js

```diff
@@ -7,15 +7,22 @@ import bundlewatchApi from '.'
 
 describe(`bundlewatch Node API`, () => {
     it('Works with basic options', async () => {
-        const result = await bundlewatchApi({
-            files: [
-                {
-                    path: './__testdata__/*.jpg',
-                    maxSize: '100kB',
-                },
-            ],
-            defaultCompression: 'none',
-        })
+        const result = await bundlewatchApi(
+            {
+                files: [
+                    {
+                        path: './__testdata__/*.jpg',
+                        maxSize: '100kB',
+                    },
+                ],
+                defaultCompression: 'none',
+            },
+            {
+                error: () => {},
+                pass: () => {},
+                fail: () => {},
+            },
```

Comment:
It seems to be a bad developer experience if we absolutely need to define `error`, `pass`, and `fail`, here even if they are empty.

I tried to remove them, but then it crashed. I would consider this as a breaking change if we were to roll this out as this is changing the required parameters for BundleWatch API.

Would that make sense to default them to `() => {}` so we could skip providing them.